lib/routes/emails.js

```diff
@@ -491,12 +494,15 @@ module.exports = (log, db, mailer, config, customs, push) => {
         const sessionToken = request.auth.credentials
         const uid = sessionToken.uid
 
-        if (! features.isSecondaryEmailEnabled(sessionToken.email)) {
-          return reply(error.featureNotEnabled())
-        }
+        return db.account(uid)
+          .then((account) => {
 
-        db.accountEmails(uid)
```

Comment:
> seems like we'd benefit from having a single canonical place to get any email address associated with the account

`db.accountRecord(email)` is capable of that. It just so happens that I also needed the accounts original email used in creation, to feature flag this. `db.accountEmails(uid)` is useful when you just need emails and didn't need any other account information.

> but is there an argument for joining with the accounts table in the accountEmails

This was done because the original account object contained all the emails (only one). This kept that contract but put them in `account.emails`.